test/widget-tests.py

```diff
@@ -48,7 +48,7 @@ def test_select_movie():
 
 def test_choose_species():
     widget = kso_widgets.choose_species(db_connection=pp.db_connection)
-    assert widget.value == ("Banded weedfish",)
+    assert widget.value == ("Nothing here",)
```

Comment:
Why did this value change?

Reply:
It seems that "Nothing here" was added later. Before it was also "Nothing here" but then later it wasn't an option anymore. So I think there was some change in the underlying csv.

Reply:
Okej :)
As long that we are sure that this is the 'correct' return now for the test.